src/components/toastNotificationsList/toastNotificationsList.js

```diff
@@ -21,46 +21,41 @@ class ToastNotificationsList extends React.Component {
   };
 
   render() {
-    const { toasts, paused } = this.props;
+    const { toasts } = this.props;
+    const timeout = 8000;
 
     return (
-      <ToastNotificationList>
+      <AlertGroup isToast className="qpc-alert-group">
```

Comment:
required:
update the css className to `quipucords-alert-group`, or attempt something fancy `quipucords-toast-notifications__alert-group` in the event we need to add classes